PSICover/PSICover.cs

```diff
@@ -182,12 +187,61 @@ class Analyzer {
             """;
          html = html.Replace ("\u00ab", "&lt;").Replace ("\u00bb", "&gt;");
          File.WriteAllText (htmlfile, html);
+         summaryList.Add ((file, hitCount, blocks.Count - hitCount, Math.Round (100.0 * hitCount / blocks.Count, 1)));
+         Summary (summaryList);
+
       }
       int cBlocks = mBlocks.Count, cHit = hits.Count (a => a > 0);
       double percent = Math.Round (100.0 * cHit / cBlocks, 1);
       Console.WriteLine ($"Coverage: {cHit}/{cBlocks}, {percent}%");
    }
 
+   void Summary (List<(string filename, int hit, int unhit, double percent)> summary) {
+      string data = "";
+      foreach (var result in summary.OrderBy (s => s.percent)) {
+         data += $"""

```

Comment:
It is better to use string.Join or StringBuilder instead of concating two strings.